scripts/verify-registry-install.sh

```diff
@@ -40,12 +41,21 @@ fi
 echo "Done with npm installation test"
 popd
 
+# installs package locally (yarn 1 struggles with registry authentication requirements)
+mkdir okta-auth-js
+cd okta-auth-js
+wget --header="Authorization: Bearer ${NPM_TOKEN}" ${published_tarball} -O ${artifact_version}.tgz
+ls -al
+cd ..
```

Comment:
Consider `pushd`/`popd` instead of `cd`

Reply:
Or put the commands within parenthesis, such as:
```suggestion
(
  cd okta-auth-js
  wget --header="Authorization: Bearer ${NPM_TOKEN}" ${published_tarball} -O ${artifact_version}.tgz
  ls -al
)
```